sample_templates/kinesis.yml.js

```diff
@@ -0,0 +1,32 @@
+# This is an example template file for getting metrics for a kinesis stream.
+# If you want an example of a kinesis application look at kinesisapp.yml.js.
```

Comment:
I added that on a previous commit. It should be under sample_templates. I wanted to add this example too for people who want to get overall kinesis stream metrics (that are independent of kinesis application collectors)